.gitignore

```diff
@@ -58,9 +58,9 @@ docs/examples/
 docs/errors.txt
 docs/images/auto-generated
 **/myscreenshot.png
-api-autogen/
+_autosummary
 
-docs/external_examples.rst
+docs/getting-started/external_examples.rst
```

Comment:
So this wasn't the real issue, sorry.

Now after `make clean` I still see this:
```
$ git status
HEAD detached at upstream/doc/autosummary
Changes not staged for commit:
  (use "git add/rm <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	deleted:    docs/getting-started/external_examples.rst
```

So if I understand correctly `docs/getting-started/external_examples.rst` is in the index. So either
1. it shouldn't be removed by `make clean`, or
2. it should be `git rm`med.

Or is there another possibility I'm missing?

Reply:
external_examples is built automatically by `make_external_gallery.py`.  I'll add it back in our `gitignore`.

Reply:
and it should be git removed.

Reply:
doing that...

Reply:
I think you already added it back? The problem seems to be that someone `git add`ed and committed that file.